content/docs/projects/trust.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Trust
+description: 'Distributing Trust Bundles in Kubernetes'
+---
+
+## Distributing Trust Bundles in Kubernetes
+
+trust is an operator for distributing trust bundles across a Kubernetes cluster.
+trust is designed to complement
+[cert-manager](https://github.com/cert-manager/cert-manager) by enabling services to
+trust X.509 certificates signed by Issuers, as well as external CAs which may
+not be known to cert-manager at all.
+
+## Usage
+
+trust ships with a single cluster scoped `Bundle` resource. A Bundle represents
+a set of data that should be distributed and made available across the cluster.
+There are no constraints on what data can be distributed.
+
+The Bundle gathers and appends trust data from a number of `sources` located in
+the trust namespace (where the trust controller is deployed), and syncs them to
+a `target` in every namespace.
+
+A typical Bundle looks like the following:
+
+```yaml
+apiVersion: trust.cert-manager.io/v1alpha1
+kind: Bundle
+metadata:
+  name: my-org.com
+spec:
+  sources:
+  # A Secret in the trust namespace created via a cert-manager Certificate
+  - secret:
+      name: "my-db-tls"
+      key: "ca.crt"
+  # A ConfigMap in the trust namespace
+  - configMap:
+      name: "my-org.net"
+      key: "root-certs.pem"
+  # An In Line
+  - inLine: |
+      # my-org.com CA
+      -----BEGIN CERTIFICATE-----
+      MIIC5zCCAc+gAwIBAgIBADANBgkqhkiG9w0BAQsFADAVMRMwEQYDVQQDEwprdWJl
+      ....
+      0V3NCaQrXoh+3xrXgX/vMdijYLUSo/YPEWmo
+      -----END CERTIFICATE-----
+  target:
+    # Data synced to the ConfigMap `my-org.com` at the key `root-certs.pem` in
+    # every namespace.
+    configMap:
+      key: "root-certs.pem"
+```
+
+Bundle currently supports the source types `configMap`, `secret` and `inLine`,
+and target type `configMap`.
+
+---
+
+## Installation
+
+First, install [cert-manager](https://cert-manager.io/docs/installation/) to the
+cluster, and then the trust operator. It is advised to run the trust operator in
+the `cert-manager` namespace.
+
+```bash
+$ helm repo add jetstack https://charts.jetstack.io --force-update
+$ helm upgrade -i -n cert-manager cert-manager jetstack/cert-manager --set installCRDs=true --wait --create-namespace
+$ helm upgrade -i -n cert-manager cert-manager-trust jetstack/cert-manager-trust --wait
```

Comment:
suggestion (non-blocking): I didn't spot this the first time around but the `$` prefix on these commands means the copy-paste button we gained on code-blocks with the new site, which might actually be pretty useful since I think people could copy these verbatim. That doesn't need to block this PR though of course!